src/app/core/audit/list-audit-entries/list-audit-entries.component.ts

```diff
@@ -78,11 +78,11 @@ export class ListAuditEntriesComponent extends AbstractPageableDataComponent<any
 
 	dateRangeFilter: any;
 
-	queryStartDate: Date = utc()
-		.subtract(1, 'days')
-		.toDate();
+	queryStartDate: Date = DateTime.now()
+		.toUTC(1440)
+		.toJSDate();
```

Comment:
This adding vs. substracting.  Also recommend using luxon's minus function vs. the offset option on `toUTC`.

```ts
DateTime.now().minus({ days: 1 }).toUTC();
```

The minus option makes the intent clearer and is more correct.  If you convert both to ISO string format, the results are as follows:

utc offset:
2021-08-04T23:59:59.999-24:00

minus:
2021-08-04T23:59:59.999Z